backend/app/db_accessors/accessor.js

```diff
@@ -0,0 +1,53 @@
+/* This is the accessor file. It connects to the database and 
+retrieves the requested information from the database. This is 
+done using MongoDB functions. */
```

Comment:
MongoDB, specifically Mongoose functions, but I think it's fine to leave it here since we don't need to explain what the Mongoose ODM is